client/js/components/map/admin/LayerSettings.vue

```diff
@@ -395,29 +395,33 @@ export default {
       }
 
       const url = this.handleUrlParams(this.url)
+      const hasWMTSType = url.toLowerCase().includes('wmts')
       let parser = null
-      dpRpc('map.get_capabilities', { url })
+
+      externalApi(url)
         .then(response => {
-          this.serviceType = (response.data['0'].result.type === 'wmts') ? 'wmts' : 'wms'
+          this.serviceType = hasWMTSType ? 'wmts' : 'wms'
```

Comment:
why is the "old way" not working anymore?

Reply:
because the response has changed, we no longer receive the expected type in a response as it was before (through the BE). we receive only an xml document. So, we need to find out alternative methods to determine the type. I think that the URL is a good option since it contains the type information

Reply:
one should not work when unconcentrated. :facepalm: